src/elements/hx-modal/index.spec.js

```diff
@@ -0,0 +1,154 @@
+                const component = /** @type { HXModalElement } */ await fixture(template);
+                const name = component.slot;
+
+                if ( name !== null ) {
+                    expect(name).to.be.equal('');
+                } else {
+                    expect(name).to.be.null;
+                }
+            });
+        });
+
+        describe('verify Shadow DOM markup', () => {
+
+            it('markup should contain a #hxBackdrop <div>', async () => {
+                const elSelector = 'div#hxBackdrop';
+                const id = 'hxBackdrop';
+                const component = /** @type { HXModalElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+
+            it('markup should contain a #hxModal <div>', async () => {
+                const elSelector = 'div#hxModal';
+                const id = 'hxModal';
+                const component = /** @type { HXModalElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+
+            it('markup should contain a #hxClose button', async () => {
+                const elSelector = 'button#hxClose';
+                const id = 'hxClose';
+                const component = /** @type { HXModalElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+
+            it('markup should contain a #hxClose button with a times type <hx-icon>', async () => {
+                const elSelector = 'button#hxClose > hx-icon';
+                const iconType = 'times';
+                const component = /** @type { HXModalElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryType = query.type;
+
+                expect(queryType).to.equal(iconType);
+            });
+
+            it('markup should contain a #hxContent <div>', async () => {
+                const elSelector = 'div#hxContent';
+                const id = 'hxContent';
+                const component = /** @type { HXModalElement } */ await fixture(template);
+                const shadow = component.shadowRoot;
+                const query = shadow.querySelector(elSelector);
+                const queryId = query.id;
+
+                expect(queryId).to.equal(id);
+            });
+        });
+    });
+
+    describe('test getters and setters', () => {
```

Comment:
Please add a second `it()` test to assert the `aria-hidden` attribute, and it's value.

Reply:
Changes have been done, please review.